src/dltuiutils.cpp

```diff
@@ -11,9 +11,11 @@ QColor DltUiUtils::optimalTextColor(QColor forColor)
     int d = 0;
 
     // Counting the perceptive luminance - human eye favors green color...
-    double a = 1 - ( 0.299 * forColor.redF() + 0.587 * forColor.green() + 0.114 * forColor.blue())/255;
+    int a = ( 0.299 * forColor.red() + 0.587 * forColor.green() + 0.114 * forColor.blue());
 
-    if (a < 0.5)
+    qCritical() << forColor <<" " <<  a;
```

Comment:
There is an build error in Windows:

./../src/dltuiutils.cpp:16:15: error: invalid use of incomplete type ‘class QDebug’

qCritical should also not be called everytime, only in error situations.

Reply:
This has already been fixed, i do not see this anymore